packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -26,33 +54,86 @@ export type PanelPadding = 'none' | 'md';
  * @internal
  */
 export const PanelChrome: React.FC<PanelChromeProps> = ({
-  title = '',
-  children,
   width,
   height,
+  children,
   padding = 'md',
+  title = '',
+  titleItems = [],
+  menu,
+  // dragClass,
+  hoverHeader = false,
+  // loadingState,
+  // states = [],
   leftItems = [],
 }) => {
   const theme = useTheme2();
   const styles = useStyles2(getStyles);
 
-  const headerHeight = getHeaderHeight(theme, title, leftItems);
+  const headerHeight = !hoverHeader ? getHeaderHeight(theme, title, leftItems) : 0;
   const { contentStyle, innerWidth, innerHeight } = getContentStyle(padding, theme, width, headerHeight, height);
 
   const headerStyles: CSSProperties = {
     height: headerHeight,
   };
-
+  const itemStyles: CSSProperties = {
+    minHeight: headerHeight,
+    minWidth: headerHeight,
+  };
   const containerStyles: CSSProperties = { width, height };
 
+  const handleMenuOpen = () => {};
+
+  const hasHeader = title || titleItems.length > 0 || menu;
+
   return (
     <div className={styles.container} style={containerStyles}>
-      <div className={styles.header} style={headerStyles}>
-        <div className={styles.headerTitle}>{title}</div>
-        {itemsRenderer(leftItems, (items) => {
-          return <div className={styles.leftItems}>{items}</div>;
-        })}
-      </div>
+      {hasHeader && !hoverHeader && (
+        <div className={styles.headerContainer} style={headerStyles} data-testid="header-container">
+          {title && (
+            <div title={title} className={styles.title}>
+              {title}
+            </div>
+          )}
+
+          {titleItems.length > 0 && (
+            <div className={styles.items} data-testid="title-items-container">
+              {titleItems
+                .filter((item) => isIconName(item.icon))
+                .map((item, i) => (
+                  <div key={`${item.icon}-${i}`} className={styles.item} style={itemStyles}>
+                    {item.onClick ? (
+                      <IconButton tooltip={item.tooltip} name={item.icon} size="sm" onClick={item.onClick} />
+                    ) : (
+                      <Tooltip content={item.tooltip ?? ''}>
+                        <Icon name={item.icon} size="sm" />
+                      </Tooltip>
+                    )}
+                  </div>
+                ))}
+            </div>
+          )}
+
+          {menu && (
+            <Dropdown overlay={menu} placement="bottom">
+              <div className={cx(styles.item, styles.menuItem, 'menu-icon')} data-testid="menu-icon" style={itemStyles}>
+                <IconButton
+                  ariaLabel={`Menu for panel with ${title ? `title ${title}` : 'no title'}`}
+                  tooltip="Menu"
+                  name="ellipsis-v"
+                  size="sm"
+                  onClick={handleMenuOpen}
```

Comment:
Switch this to the normal button with variant secondary, fill="text" 

  <Button
                  aria-label={`Menu for panel with ${title ? `title ${title}` : 'no title'}`}
                  variant="secondary"
                  fill="text"
                  tooltip="Menu"
                  icon="ellipsis-v"
                  size="sm"
                  onClick={handleMenuOpen}
                />